test/adaptive_load/utility.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
```

Comment:
Can we give this library a name that is a bit less generic? File named like "utility" or "tools" will simply attract anything (since anything is a utility) and may become hard to manage.

Consider: minimal_output.(h|cc)

Reply:
Done.